owasp-top10-2017-apps/a5/tictactoe/README.md

```diff
@@ -139,7 +142,7 @@ And see his private statistics:
 
   
 
-Using another user `bender` and the Burp Suite Tool, we can see the API request details when accessing _localhost:10005/statistics_. As seen below, to get the user statistics data, an ajax request is made passing the username as a URL parameter:
+Using another user `bender` and the Burp Suite Tool, we can see the API request details when accessing _localhost:10005/statistics_. As seen below, to get the user statistics data, an AJAX request is made passing the username as a URL parameter:
```

Comment:
We could use a link to this URL instead of an italic font. What do you think? 🙃

Reply:
Done!